components/centraldashboard/app/api.ts

```diff
@@ -70,18 +69,42 @@ export class Api {
             })
         .get(
           '/dashboard-links',
-          async (_: Request, res: Response) => {
+          async (req: Request, res: Response) => {
+            const acceptlanguage = req.header('accept-language');
             const linksData = await this.k8sService.getDashboardLinks();
-            let links = {};
+            let langLinks = {};
             try {
-              links=JSON.parse(linksData.data["links"]);
+              const links = JSON.parse(linksData.data["links"]);
+              const lang = this.resolveLanguage
+              (
```

Comment:
Keep parenthesis on same line as method ref so it doesn't look like we're just referencing a property of `this`.